src/libs/blackboard/net/interface_proxy.h

```diff
@@ -72,7 +73,9 @@ class BlackBoardInterfaceProxy : public InterfaceMediator, public MessageMediato
 	inline unsigned int
 	next_msg_id()
 	{
-		return ((instance_serial_ << 16) | next_msg_id_++);
```

Comment:
is it necessary to avoid collisions with message ids from other serials? I am not to familiar with the whole message handling in fawkes, but i would have thought the unique sender uuid would now make this obsolete.

Reply:
No I don't think so. I wasn't sure at first, which is why I added the TODO. Removed now.